includes/class-wc-subscriptions-email-notifications.php

```diff
@@ -88,14 +88,13 @@ public static function set_notification_settings_update_time() {
 
 		// Shortcut to unschedule all notifications more efficiently instead of processing them subscription by subscription.
 		if ( ! self::notifications_globally_enabled() ) {
-			\WC_Subscriptions_Core_Plugin::instance()->notifications_scheduler->unschedule_all_notifications();
-
-			$message = __( 'Unscheduling all notifications now.', 'woocommerce-subscriptions' );
+			as_unschedule_all_actions( null, [], 'wcs_customer_notifications' );
```

Comment:
I think it would be better if the group name weren't hardcoded here as a string. That was one of the reasons why I decided to use the method instead of calling the `as_...` function directly. Other option would be to make the group public. Not strongly against that either. What do you think?